server/driver/history.h

```diff
@@ -47,6 +48,19 @@ class history
 		// Discard outdated data, packets could be reordered
 		if (not data.empty())
 		{
+			// keep only one sample if the clock_offset is unreliable
+			if (not offset)
```

Comment:
All the paths that lead to `add_sample` check if clock offset is valid in `wivrn_session`.

We should never reach this place, if we want to keep a single sample with unstable clock, we should remove the checks.